docs/technical-details/release-notes/4.tucker/README.md

```diff
@@ -2,13 +2,15 @@
 
 HarperDB version 4 ([Tucker release](./tucker.md)) represents major step forward in database technology. This release line has ground-breaking architectural advancements including:
 
-## [4.0](./4.0.0.md)
-* New clustering technology that delivers robust, resilient and high-performance replication
-* Major storage improvements with highly-efficient adaptive-structure modified MessagePack format, with on-demand deserialization capabilities
+## [4.4](./4.4.0.md)
+* Native replication (codename "Plexus") which is faster, more efficient, secure, and reliable than the previous replication system and provides provisional sharding capabilities with a foundation for the future
```

Comment:
@jcohen-hdb  @kriszyp What are your thoughts

 Native replication (codename "Plexus"), which is a faster, more efficient, and secure replication system that improves reliability over 4.3. It also introduces provisional sharding capabilities, laying the groundwork for more advanced scaling in the future.